apps/researcher/src/app/[locale]/page.tsx

```diff
@@ -32,17 +33,39 @@ import {ElementType} from 'react';
 // Revalidate the page every n seconds
 export const revalidate = 60;
 
-interface FacetProps {
+interface FilterSetting {
   name: keyof SearchResult['filters'];
   searchParamType: SearchParamType;
+}
+
+interface FacetComponent {
+  name: string;
   Component: ElementType;
```

Comment:
Lowercase `c`, `component`?

Reply:
`ElementType`s need to be uppercase to be valid JSX.

Reply:
Can you this as a comment to the code? Then future readers understand why we're breaking the lowercase convention in interface props.